tests/cypress/support/commands.js

```diff
@@ -132,6 +134,33 @@ Cypress.Commands.add('checkUserStatuses', (authKey, userName, staffStatus, super
     });
 });
 
+Cypress.Commands.add('deletingCreatedTasks', (response, tasksToDelete) => {
+    const authKey = response.body.key;
+    cy.request({
+        url: '/api/v1/tasks?page_size=all',
+        headers: {
+            Authorization: `Token ${authKey}`,
+        },
+    }).then((_response) => {
+        const responceResult = _response.body.results;
+        for (const task of responceResult) {
+            const taskId = task.id;
+            const taskName = task.name;
+            for (const taskToDelete of tasksToDelete) {
+                if (taskName === taskToDelete) {
+                    cy.request({
+                        method: 'DELETE',
+                        url: `/api/v1/tasks/${taskId}`,
+                        headers: {
+                            Authorization: `Token ${authKey}`,
+                        },
+                    });
+                }
+            }
+        }
+    });
+});
```

Comment:
I have similar comments for this code like above

Reply:
Done